build.py

```diff
@@ -1,26 +1,31 @@
 from sklearn.datasets import load_iris
 from sklearn.model_selection import train_test_split
 from sklearn.linear_model import LogisticRegression
-
-from skl2onnx import convert_sklearn
-from skl2onnx.common.data_types import FloatTensorType
+from hummingbird.ml import convert
+import shutil
+import zipfile
+import os
 
 # prepare the train and test data
 iris = load_iris()
 X, y = iris.data, iris.target
 X_train, X_test, y_train, y_test = train_test_split(X, y)
 
-# train a model
+# train the model - using logistic regression classifier
 model = LogisticRegression(max_iter=5000)
 model.fit(X_train, y_train)
 
-# convert the model to ONNX
-initial_types = [
-  ('input', FloatTensorType([None, 4]))
-]
-
-onnx_model = convert_sklearn(model, initial_types=initial_types)
+# use hummingbird.ml to convert sklearn model to torchscript model (torch.jit backend)
+torch_model = convert(model, 'torch.jit', test_input=X_train, extra_config={})
 
 # save the model
-with open("iris.onnx", "wb") as f:
-  f.write(onnx_model.SerializeToString())
+torch_model.save('iris.torch')
+
+# extract the TorchScript binary payload
+with zipfile.ZipFile('iris.torch.zip') as z:
```

Comment:
I'd use the `from zipfile import ZipFIle` above so that you can just say `with ZipFile('...') as z:`.

Reply:
Specifically because zipfile.ZipFile feels pretty redundant.

Reply:
Oh, I like that!